db_test.go

```diff
@@ -2635,3 +2635,39 @@ func TestCompactL0OnClose(t *testing.T) {
 		}
 	})
 }
+
+func TestConcurrent(t *testing.T) {
```

Comment:
could you name it better? Something like TestCloseDBWhileReading

Reply:
ok